tap_square/streams.py

```diff
@@ -169,6 +169,22 @@ def sync(self, client, start_time, bookmarked_cursor): #pylint: disable=no-self-
             yield page, cursor
 
 
+class Shifts:
+    tap_stream_id = 'shifts'
+    key_properties = ['id']
+    replication_method = 'INCREMENTAL' # Maybe 'FULL_TABLE'
+    valid_replication_keys = ['updated_at']
+    # TODO We cannot query the API with an updated at
+    # - There's a `filter` in the `body` we can use, but that will filter on the start time of the shift
+    #   - Strategy 1: Always query with the start time for the tap config, ordering by `updated_at` and emit the
```

Comment:
Should we ask the team for their thoughts on this? If you already wrote the code it's fine, but otherwise I would prefer to avoid the work if the tap is not used by so many people.

Reply:
I'm partially wrote the code. I didn't do the part where we filter with the start time in the config, it would take a trivial amount of work to do so.

The tap currently orders by `updated_at` and will only write records with `updated_at` `>=` the bookmark

Reply:
We can ask the team and address the changes during the refactor